doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -0,0 +1,126 @@
+How to Setup Moveit2 Docker Containers in Ubuntu
+=================================================
+This guide will provide a walkthrough on how to get docker container with Moveit2 dependencies setup quickly.
+Provided in this guide is a script that will get you up and running in Moveit quickly!
+This guide is intended for people who would like to have a separate environment for working with Moveit up and running quickly \
+without having to do much configuring. In this guide, we will be setting up a ROS2 Galactic environment.
+
+Learning Objectives
+-------------------
+
+- How to setup a docker environment using the provided script
+
+Requirements
+------------
+
+- Ubuntu 20.04
+- `Docker Installation for Ubuntu <https://docs.docker.com/engine/install/ubuntu/>`_
+
+Steps
+-----
+
+1. Open a terminal session and create an empty text file.
+::
+  mkdir -p ~/Docker/scripts
+  cd ~/Docker/scripts/
+  gedit start-docker.sh
```

Comment:
I think it's better if we make people use this script as functions in their bashrc